mne/io/reference.py

```diff
@@ -97,11 +128,16 @@ def _apply_reference(inst, ref_from, ref_to=None):
                 'using the apply_proj() method function before the desired '
                 'reference can be set.'
             )
+
     for i in projs_to_remove:
         del inst.info['projs'][i]
```

Comment:
I think there should at least be a warning. Maybe at this stage, but maybe also later - e.g., there could be a block for plotting functions for channels that have different references.

Though maybe it'd be sufficient to just warn here.